src/test/java/net/sf/jabref/logic/exporter/ModsExportFormatTestFiles.java

```diff
@@ -54,13 +56,19 @@
 
     @Parameters(name = "{0}")
     public static Collection<String> fileNames() throws Exception {
-        try (Stream<Path> stream = Files.list(Paths.get(ModsExportFormatTestFiles.class.getResource("").toURI()))) {
-            //            stream.forEach(n -> System.out.println(n));
+        String packageName = MSBibExportFormatTestFiles.class.getPackage().getName();
```

Comment:
Same as previous, should be `ModsExportFormatTestFile.class.getPackage().getName()`.